tests/unit/deploy/test_planner.py

```diff
@@ -365,6 +365,15 @@ def sweeper(self):
     def function_resource(self):
         return create_function_resource('myfunction')
 
+    def one_deployed_lambda_function(name='myfunction', arn='arn'):
+        return {
+            'resources': [{
+                'name': 'myfunction',
```

Comment:
Did you mean to use the `name` and `arn` as values here?

Reply:
Yeah the `name` variable needs to be used here.

Reply:
Yeah, good catch.